src/distgen.cpp

```diff
@@ -80,13 +80,7 @@ static size_t get_tcount() {
 
 	if (tc > 0) return tc;
 
-#ifdef _OPENMP
```

Comment:
That looks broken. Is that function still used?
If not -> remove it
If yes -> it should return the maximum number of threads.